replica/table.cc

```diff
@@ -1583,13 +1583,17 @@ table::try_flush_memtable_to_sstable(compaction_group& cg, lw_shared_ptr<memtabl
 void
 table::start() {
     start_compaction();
+    if (_schema->memtable_flush_period() > 0 && this_shard_id() == 0) {
```

Comment:
Why only shard 0? There are memtables on all shards.

Reply:
please correct me if Im wrong but I thought that `table::flush()` flushed memtables for all shards, isn't it?

Reply:
No, `replica::table::flush()` only flushes the memtables on the local shard.
We have `replica::database::flush_table_on_all_shards()` which flushes a given table on all shards, but this is not available from the table objects, so you have to setup the timer on all shards.